.travis.yml

```diff
@@ -18,6 +21,13 @@ matrix:
       env: SYMFONY_VERSION="4.2.*"
     - php: 7.0
       env: SYMFONY_VERSION="4.3.*"
+    - php: 7.0
+      env: SYMFONY_VERSION="4.4.*"
+    - php: 7.0
+      env: SYMFONY_VERSION="5.0.*"
+    # Symfony >= 5.0 PHP requirement is ^7.2.5
+    - php: 7.1
+      env: SYMFONY_VERSION="5.0.*"
 
 sudo: false
 
```

Comment:
Looking at our composer.json, I noticed that we need the same line for `symfony/translation` as for `symfony/workflow` in the `before_install` section so the translation component is required at the same version during the test suite

Reply:
`symfony/translation` is a dev dependency used for test purposes. Why you consider we should tie its version with `symfony/workflow`?

Reply:
It is a dev dependency because sonata requires it anyway, but we declared it explicitly to be able to require precise version.
We want to be able to work with any symfony version, when working on a symfony 4.2 project we should have both workflow and translation at this version (I hope people are doing it like this at least).
This is why we must require a specific version during tests

Reply:
IMO, when you work with decoupled components, the only concern is to satisfy the constraints declared for them. So, if `symfony/workflow:5.0.0` can coexist in the same project with `symfony/translation:4.4.0` I don't see any value on forcing both packages to the same version while testing.
I've pushed a new commit installing `symfony/translation` at the same version of `symfony/workflow` and the result is a [non installable set of packages](https://travis-ci.org/yokai-php/sonata-workflow/jobs/656050967#L265).

Reply:
Looks like sonata admin / symfony 5.x compatibility is not ready yet : 

**create symfony 5.x empty project**

```
composer create-project symfony/skeleton:^5.0 symfony-5-plus-sonata
cd symfony-5-plus-sonata
```

**require sonata admin stable version**

```
composer require sonata-project/admin-bundle

Using version ^3.61 for sonata-project/admin-bundle
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for sonata-project/admin-bundle ^3.61 -> satisfiable by sonata-project/admin-bundle[3.61.0].
    - sonata-project/admin-bundle 3.61.0 requires symfony/console ^4.3 -> satisfiable by symfony/console[4.3.x-dev, 4.4.x-dev, v4.3.0, v4.3.0-BETA1, v4.3.0-BETA2, v4.3.0-RC1, v4.3.1, v4.3.10, v4.3.11, v4.3.2, v4.3.3, v4.3.4, v4.3.5, v4.3.6, v4.3.7, v4.3.8, v4.3.9, v4.4.0, v4.4.0-BETA1, v4.4.0-BETA2, v4.4.0-RC1, v4.4.1, v4.4.2, v4.4.3, v4.4.4] but these conflict with your requirements or minimum-stability.
```

**require sonata admin 4.x version**

```
composer require sonata-project/admin-bundle:4.x-dev@dev

Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for sonata-project/admin-bundle 4.x-dev@dev -> satisfiable by sonata-project/admin-bundle[4.x-dev].
    - sonata-project/admin-bundle 4.x-dev requires symfony/console ^4.3 -> satisfiable by symfony/console[4.3.x-dev, 4.4.x-dev, v4.3.0, v4.3.0-BETA1, v4.3.0-BETA2, v4.3.0-RC1, v4.3.1, v4.3.10, v4.3.11, v4.3.2, v4.3.3, v4.3.4, v4.3.5, v4.3.6, v4.3.7, v4.3.8, v4.3.9, v4.4.0, v4.4.0-BETA1, v4.4.0-BETA2, v4.4.0-RC1, v4.4.1, v4.4.2, v4.4.3, v4.4.4] but these conflict with your requirements or minimum-stability.
```

So, Symfony 5 compatibility have no sense for the moment : **we can remove it from testing matrix**

Reply:
`symfony/skeleton` is "a minimal Symfony project recommended to create bare bones applications". It already defines a set of packages which are not compatible with `sonata-project/admin-bundle`.
This doesn't mean that a project using its own set of packages can not use different versions if they are allowed.
By instance, in my project `my/project`, if I decide to bump `symfony/workflow` to `^5.0`, the only package preventing this is `yokai/sonata-workflow`:
```
$ composer why-not symfony/workflow:5
my/project       3.32.x-dev  requires  symfony/workflow (^4.4)       
yokai/sonata-workflow  v0.2.0      requires  symfony/workflow (^3.2|^4.0)
```

Reply:
`symfony/skeleton` is fully compatible with `sonata-project/admin-bundle`, if you are using `^4.0` versions.
So if you `composer create-project symfony/skeleton:^4.0 symfony-4-plus-sonata`, then `composer require sonata-project/admin-bundle` works like a charm.

In your project, you have some `symfony 4.x.y` and some `symfony 5.x.y` ?

Let's figure out a way for this.
Remove the symfony/translation version requirement I asked previously, I will merge it like this.

Reply:
>In your project, you have some symfony 4.x.y and some symfony 5.x.y ?

Yes, since I have a big tree of dependencies, I'm upgrading packages in a long term process.

>Let's figure out a way for this.
Remove the symfony/translation version requirement I asked previously, I will merge it like this.

Thank you so much. I've pushed a commit removing the dev requirement for "symfony/translation".